### DISCUSSION TITLE
欢迎来到Ed3Academy讨论室！!

### DISCUSSION THREAD

**Discussion by luuuuo** [Ed3Academy/blockchain-courses#5] [公告]
## 👋 Welcome!
来都来了，随便说点什么吧！
我们将Github Discussion用作与社区其他成员联系的地方。 我们希望您：

问你想知道的问题。
分享意见。
您可以在这里发帖、灌水、交友与其他社区成员互动。
非常希望您能留下点意见或者建议，共同把Ed3Academy建设得更加完善！💪💪💪
<!--

**Comment by tianxiuali** [comment]
🛋️沙发
